### ISSUE TITLE
NTR: Creatine / creatinine ratio high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8799]
New term request Creatine / creatinine ratio high in urine (segal_020805190805):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000127
- label: Elevated urinary creatine/creatinine ratio
- parents: [Abnormal urine metabolite level (HP:0033354)](https://hpo.jax.org/app/browse/term/HP:0033354)
- pmid: PMID:27096572
- def: The ratio of creatine to creatinine concentration in the urine is above the upper limit of normal.
- comment: This finding is suggestive of a disorder of creatine metabolism.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8799
- synonyms: Increased urine Cr/Crn ratio(NONE)
